{"uuid": "cbcb7d52-7566-4953-b649-c2cc3ed4b509", "children": ["d0f0da0c-ded5-46ea-9ff7-b3d733135d4f"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 128\nCurrent browser version is 130.0.6723.70 with binary path C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\n\tGetHandleVerifier [0x00007FF78D3FB632+29090]\n\t(No symbol) [0x00007FF78D36E6E9]\n\t(No symbol) [0x00007FF78D22B1CA]\n\t(No symbol) [0x00007FF78D26D6B2]\n\t(No symbol) [0x00007FF78D26C751]\n\t(No symbol) [0x00007FF78D26729C]\n\t(No symbol) [0x00007FF78D261E0A]\n\t(No symbol) [0x00007FF78D2B236D]\n\t(No symbol) [0x00007FF78D2B1A20]\n\t(No symbol) [0x00007FF78D2A6493]\n\t(No symbol) [0x00007FF78D2709B1]\n\t(No symbol) [0x00007FF78D271B11]\n\tGetHandleVerifier [0x00007FF78D71883D+3294125]\n\tGetHandleVerifier [0x00007FF78D764423+3604371]\n\tGetHandleVerifier [0x00007FF78D75A2E7+3563095]\n\tGetHandleVerifier [0x00007FF78D4B6F16+797318]\n\t(No symbol) [0x00007FF78D37986F]\n\t(No symbol) [0x00007FF78D375454]\n\t(No symbol) [0x00007FF78D3755E0]\n\t(No symbol) [0x00007FF78D364A7F]\n\tBaseThreadInitThunk [0x00007FFA7EE87374+20]\n\tRtlUserThreadStart [0x00007FFA7F21CC91+33]\n\n", "trace": "  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\Desktop\\Diplom\\Diplom_3\\conftest.py\", line 24, in driver\n    browser = webdriver.Chrome()\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1730211446918, "stop": 1730211452151}], "start": 1730211446918, "stop": 1730211497214}